app/views/operations/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<div id= "">
+  <p style="color: green"><%= notice %></p>
+
+  <div class= "header">
+    <%= link_to groups_path do %><%= image_tag "back.png", class:"back"%><%end%>
+    <h2>Transactions</h2>
+    <%= button_to destroy_user_session_path, method: :delete, class: "off" do %> 
+      <%= image_tag "off.png", class:"off"%>
+    <%end%>
+  </div>
+
+  <div class= "total-transactions">
+    <small class= "total-transactions">
+    <strong>Total Transactions:</strong>
+    $<%= @operations.inject(0) { |sum, t| sum += t.amount  }%> 
+    </small>
+  </div>
+
+  <div id="operations">
+    <% if @operations.any? %>
+      <% @operations.each do |operation| %>
+        <%= render operation %> 
+      <% end %>
+    <% else %>
+      <h2 style="text-align: center">No Transactions found, Add a new operation.</h2>
+    <% end %>
+  </div>
+  <div class= "new-group">
+    <%= link_to "New operation", new_group_operation_path(@operations) %>
+  </div>
```

Comment:
- [ ] Please style the transaction or `operation` index page to look like the given behance design guidelines taking note of the layout, colors, and typography as shown below 👇 
- [ ] Style the "New operation" button to take up the entire width of the page as shown in the design guideline
### Behance Design
![image](https://user-images.githubusercontent.com/46839548/227562290-ac3d8494-977d-4b8b-bcae-9e625a1494c2.png)
### Your Design
![image](https://user-images.githubusercontent.com/46839548/227563256-1d0a60b3-1e76-4a90-8a3f-9e65f1da9d0d.png)